stuffioAPI/stuffioAPI/stuffioAPI.csproj.user

```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
This file needs to be removed and added to `.gitignore`

Reply:
added to .gitignore